ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml

```diff
@@ -8,12 +8,16 @@ import StatusQ.Controls 0.1
 
 import shared 1.0
 import shared.popups 1.0
+import shared.views.chat 1.0
+
 import utils 1.0
 
 import "../views"
 import "../panels"
 
 Popup {
+    id: activityCenter
```

Comment:
"root" :)

Reply:
no pls, it takes more changes for task which is not refactoring :)

Reply:
no pls, it takes more changes for task which is not refactoring :)

Reply:
Ok, fair enough